packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2676,6 +2676,35 @@ class UnconstrainedBox extends StatelessWidget {
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=PEsY654EGZ0}
 ///
+/// {@tool dartpad --template=stateless_widget_scaffold}
+///
+/// This sample shows [FractionallySizedBox] widget that has one container
```

Comment:
This sample shows a [FractionallySizedBox] whose one child is 50% of the box's size per the width and height factor parameters, and centered within that box by the `alignment` parameter.

Reply:
thanks